vimiv/gui/thumbnail.py

```diff
@@ -446,6 +532,7 @@ def resizeEvent(self, event):
         """Update resize event to keep selected thumbnail centered."""
         super().resizeEvent(event)
         self.scrollTo(self.currentIndex())
+        self._update_icon_position()
```

Comment:
Could you elaborate why this is needed? Upon resizing the widget / window, we do have to update scrolling, as the view may have changed, but we shouldn't have to generate the thumbnails again, or am I missing something?

Noticed that this was called a lot more in testing than I thought it would due to this part.

Reply:
I honestly do not remember why I added this. It might had just been that I overzealously added it to every function I felt could have influenced the selected thumbnail. At least during testing, removing this doesn't seem to make a difference, though. So, I will do that.